src/Microsoft.ML.CodeGen/Microsoft.ML.CodeGen.csproj

```diff
@@ -0,0 +1,116 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We need to add signing stuff in here. Let's take a look at what the other project did for signing.

Reply:
Just ask @Dmitry-A , AutoML uses a different signing mechanism. So we don't have to explicitly handle that. (correct me if anything wrong)